src/components/Registration/RegPeopleAndRoles.vue

```diff
@@ -223,8 +223,8 @@ export default class RegPeopleAndRoles extends Mixins(PeopleRolesMixin) {
     // assign party type (org or person)
     this.currentOrgPerson.officer.partyType = partyType
 
-    // pre-populate Completing Party's name, email address and mailing address
-    if (roleType === RoleTypes.COMPLETING_PARTY && partyType === PartyTypes.PERSON) {
+    // pre-populate Completing Party's name, email address and mailing address only if the logged in user is not staff
+    if (roleType === RoleTypes.COMPLETING_PARTY && partyType === PartyTypes.PERSON && !this.isRoleStaff) {
```

Comment:
I think we should have a unit test checking this new functionality.

Reply:
Hi @lambert-alex ,
I am mostly backend side and finiding it difficult to how to put unit test for this. 
I checked the unit tests folder but was not able to locate tests which are related to logged in user behaviour. 
Can you point to some if they already exists ( Probably I would not have looked in detail) ?

Reply:
In my opinion, the best way to handle this (and the lack of tests elsewhere) is to create a "staff" test suite where we can mount different components and verify that they display/hide the expected elements.

As this ^^ is beyond the scope of this ticket, Ritvick, please create a Tech Debt ticket (in Entities team space) to do this.

Reply:
Created Tech debt ticket here: https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13614

Reply:
@lambert-alex OK to resolve this conversation?